pkg/scheduler/scheduler_test.go

```diff
@@ -340,7 +340,10 @@ func TestSchedulerScheduleOne(t *testing.T) {
 				st.RegisterQueueSortPlugin(queuesort.Name, queuesort.New),
 				st.RegisterBindPlugin(defaultbinder.Name, defaultbinder.New),
 			)
-			fwk, err := st.NewFramework(registerPluginFuncs, frameworkruntime.WithClientSet(client))
+			fwk, err := st.NewFramework(registerPluginFuncs,
+				frameworkruntime.WithClientSet(client),
+				frameworkruntime.WithEventRecorder(eventBroadcaster.NewRecorder(scheme.Scheme, testSchedulerName)),
```

Comment:
Can we call profile.New instead?

Reply:
> Can we call profile.New instead?

```
func NewProfile(cfg config.KubeSchedulerProfile, frameworkFact FrameworkFactory, recorderFact RecorderFactory, opts ...frameworkruntime.Option) (Profile, error)
```

I think `NewProfile()` need a parameter  `config.KubeSchedulerProfile` but not in this case.

Reply:
sg